flagging_site/blueprints/flagging.py

```diff
@@ -76,9 +79,12 @@ def index() -> str:
     
     df = latest_model_outputs()
     df = df.set_index('reach')
+
+    overridden_reaches = get_currently_overridden_reaches()
+
     flags = {
-        key: val['safe']
-        for key, val
+        reach: val['safe'] and reach not in overridden_reaches
```

Comment:
Looks good for now. But this raises an interesting question, which is: how do we communicate situations in which the model is saying "it's ok" but the cyanobacteria override is in effect and is saying "it's not ok"?

What I'm thinking is:

- In the frontpage we want the behavior here that you've specified. 
- In `model_outputs` we want a new column called "Manual Override" that is distinct from the model output.
  - We want to make clear the distinction between a model result and an override. Maybe we label outputs from the model as "Model Output" instead of merely "Output" or "Safe."
   - "Safe" should be a column that's just equivalent to `all([model_output, not manual_override])`.
- Eventually all this stuff is going to be passed through one super-function, I think. This super-function would just consolidate all the functions we have across all these pages that basically all display the same data but in different ways.  
  
  That super-function needn't have any logic about what type of output it is returning. It can simply return both the `model_output` and the `safe` at the same time, and decide during page rendering which is the value to use. (`safe` for the home page, and both for `/model_outputs` and `/api/v1/model`).

Thoughts?